bridgev2/portal.go

```diff
@@ -17,6 +17,7 @@ import (
 	"time"
 
 	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
```

Comment:
Oops, missed this, global logs are forbidden, make sure to use `log := zerolog.Ctx(ctx)` where necessary